virttest/libvirt_xml/vm_xml.py

```diff
@@ -2051,6 +2062,49 @@ def remove_feature(self, name):
             root.remove(remove_feature)
 
 
+class VMVCPUsXML(base.LibvirtXMLBase):
+
+    """
+    VCPUs tag XML class
+
+    Elements:
+        vcpus: list of dict - id, enabled, hotpluggable, order
```

Comment:
Modified to:
+    vcpus tag XML class
+
+    Elements:
+        vcpu: list of dict - id, enabled, hotpluggable, order